### PR TITLE
Add saturation and brightness to options

### PR DESCRIPTION
**Pull request by akhra**

### CODE REVIEW

File: identicon.js [hunk 1]
```diff
@@ -45,7 +47,9 @@
 
         // foreground defaults to last 7 chars as hue at 50% saturation, 70% brightness
         var hue          = parseInt(this.hash.substr(-7), 16) / 0xfffffff;
-        this.foreground  = this.options.foreground || this.hsl2rgb(hue, 0.5, 0.7);
+        var saturation  = this.options.saturation || this.defaults.saturation;
```
Comment (stewartlord) [inline identicon.js]: Nitpick: alignment of assignments is slightly off

**Review by stewartlord — CHANGES_REQUESTED** [review]
Looks great. Spacing tweaks, then it should be good to go!

File: identicon.js (hunk 1 above)
Comment (akhra) [reply identicon.js]: So it was! Good now?

**Review by stewartlord — APPROVED** [review]
Looks perfect 👌

**Pull request merged**